frontend/public/css/style.css

```diff
@@ -109,21 +109,30 @@ input[type=range] {
 
     border-radius: 8px;
 }
+.auto-presets-modal input[type=range] {
```

Comment:
They where there since this was initially the idea but I decided to replace them with the -/+ buttons as we have now. We will however add a slider for the overlap so that's why I left the styling for them here. The overlap to the UI is another sprint item thats why I didn't include this in this pullrequest.